website/site/content/docs/beta-features.md

```diff
@@ -88,3 +88,38 @@ To enable this feature, you can set the following option in your `config.yml`:
 backend:
   squash_merges: true
 ```
+
+## Customizing Commit Messages
+You can customize the commit messages generated by Netlify CMS with templates in the `backend` option of your `config.yml`.
+
```

Comment:
To me, "customize commit messages" implies editing individual messages, rather than the templates that generate them. I think it might be clearer if the above two lines become something more like:
> ## Commit Message Templates
> You can customize the templates used by Netlify CMS to generate commit messages by setting the `commit_messages` option under `backend` in your `config.yml`.